loader.lua

```diff
@@ -28,10 +31,13 @@ SIDEBOARD_POSITION_OFFSET = {-1.47, 0.2, 0.1286}
 COMMANDER_POSITION_OFFSET = {0.7286, 0.2, -0.8257}
 TOKENS_POSITION_OFFSET = {-0.7286, 0.2, -0.8257}
 
+DEFAULT_CARDBACK = "https://gamepedia.cursecdn.com/mtgsalvation_gamepedia/f/f8/Magic_card_back.jpg?version=0ddc8d41c3b69c2c3c4bb5d72669ffd7"
+
 ------ GLOBAL STATE
 lock = false
 playerColor = nil
 deckSource = nil
+advanced = false
 
```

Comment:
I think we probably want to track the custom cardback in global state as well, so that it doesn't disappear when the user closes the advanced panel.
```suggestion
customCardBack = ""
```

Reply:
Fixed, I had it as a global but buried inside the function that used it and not up here